src/components/ec-dropdown-search/types.ts

```diff
@@ -0,0 +1,33 @@
+import type { StyleValue } from 'vue';
+
+import type { TooltipOptions } from '../../directives/ec-tooltip/types';
+import type { ZIndexLevel } from '../../enums';
+import type { PopoverProps } from '../ec-popover/types';
+
+export interface DropdownItem<TValue = string> {
```

Comment:
```suggestion
export interface DropdownSearchItem<TValue = string> {
```

Reply:
I was thinking about it too but this is shared between ec-dropdown and ec-dropdown-search, so because the ec-dropdown is used more often, it's going to be better to name it DropdownItem. I can also try to make an alias and declare `DropdownItem<TValue> = DropdownSearchItem<TValue>`.

Reply:
I would use prefixes consistently. So here `DropdownSearch` and in normal dropdown just `Dropdown`. It also reflects the dependency `dropdown` on `dropdown-search` if we use `DropdownSearchItem` in `dropdown` component.